shm/Player.cpp

```diff
@@ -9,8 +9,8 @@ Player::Player(std::unique_ptr<Ship> ship, size_t money) {
 };
 
 size_t Player::countAvailableSpace() {
-    auto occupiedSpace = std::accumulate(begin(ship_->getAllCargos()),
-                                         end(ship_->getAllCargos()),
+    auto occupiedSpace = std::accumulate(begin(getAllCargos()),
+                                         end(getAllCargos()),
```

Comment:
Actually I prefer the previous version - for me it's easier to understand that I want to get AllCargos from ship when I see `ship_->getAllCargos()`. It's just explicit way to say "get cargo from ship object". And `getAllCargos()` it's like - "okay, get cargo but from where?".